Gdip_All.ahk

```diff
@@ -2383,7 +2383,7 @@ Gdip_TextToGraphics(pGraphics, Text, Options, Font:="Arial", Width:="", Height:=
 	Gdip_DeleteFont(hFont)
 	Gdip_DeleteFontFamily(hFamily)
 
-	return _E ? _E : ReturnRC
+	return ReturnRC
```

Comment:
previously it complained about _E being undefined here (since L2375 branch was not reached)